ivy/container/random.py

```diff
@@ -1039,6 +1039,24 @@ def static_shuffle(
         -------
         ret
             A container object, shuffled along the first dimension.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([7,6,0]), \
+                              b=ivy.array([8,9,4]))
+        >>> ivy.Container.static_shuffle(x)
+        {
+            a: ivy.array([7, 0, 6]),
+            b: ivy.array([8, 4, 9])
+        }
+
+        >>> x = ivy.Container(a=ivy.array([0,0,0]), \
```

Comment:
This example with `0`s being shuffled is not interesting and should be removed as it does not display any of the function's behavior. The above example alone is sufficient.

Reply:
Done, removed this example with `0`s! 😅